ext/standard/head.c

```diff
@@ -121,6 +120,9 @@ PHPAPI int php_setcookie(zend_string *name, zend_string *value, time_t expires,
 	if (domain) {
 		len += ZSTR_LEN(domain);
 	}
+	if (samesite) {
```

Comment:
Shouldn't you check that the *samesite* value is either `Strict` or `Lax`?

Reply:
Note that `samesite` may *also* be `NULL`, here.